libadcc/make_symmetry.cc

```diff
@@ -327,4 +327,76 @@ std::shared_ptr<Symmetry> make_symmetry_operator_basis(
   return sym;
 }
 
+/**
```

Comment:
Why are these changes commented out? Are they required/will they be required for future work?